src/errors/test-run/templates.js

```diff
@@ -43,8 +43,9 @@ export default {
     [TEST_RUN_ERRORS.uncaughtErrorOnPage]: err => `
         A JavaScript error occurred on ${formatUrl(err.pageDestUrl)}.
         Repeat test actions in the browser and check the console for errors.
-        If you see this error, it means that the tested website caused it. You can fix it or disable tracking JavaScript errors in TestCafe. To do the latter, enable the "--skip-js-errors" option.
-        If this error does not occur, please write a new issue at:
+        This error is probably caused by the tested website and not by TestCafe.
```

Comment:
probably is a weasel word! on the other hand, we can't make this claim with 100% certainty. Why include this sentence at all?
```suggestion
```